usage/jsgui/src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -85,6 +85,13 @@ define([
         if (entity.config && _.size(entity.config)) result["brooklyn.config"] = entity.config;
         return result;
     }
+    function getConvertedConfigValue(value) {
+        try {
+            return $.parseJSON(value);
+        } catch (e) {
+            return value;
```

Comment:
@neykov The example you gave seems very convoluted. If you want to pass a list from a `String`, chances are you will use a comma-separated list (i.e. `1,2,3,4`) which is simpler to parse than `[1, 2, 3, 4]`, don't you think? This is actually what my client does, he uses comma-separated string to pass a list of options.

Now, I do get you point about the parsing but I think this is a use case that we will never reach because of this PR and its ability to pass a real array.